helix-lsp/src/client.rs

```diff
@@ -28,6 +28,7 @@ pub struct Client {
 }
 
 impl Client {
+    #[tracing::instrument(name = "Client::start")]
```

Comment:
I don't know if the name is necessary on any of these, tracing will use the method name by default. The module will be used as the target.

> By default, the generated span’s name will be the name of the function, the span’s target will be the current module path, and the span’s level will be INFO